m365-developer-proxy-plugins/MockResponses/GraphMockResponsePlugin.cs

```diff
@@ -4,6 +4,8 @@
 using System.Net;
 using System.Text.Json;
 using System.Text.RegularExpressions;
+using Microsoft.EntityFrameworkCore.Metadata;
```

Comment:
Are we using EntityFrmeworkCore for something or is this a leftover?

Reply:
Must be a leftover. Removed